examples/lite/README.md

```diff
@@ -0,0 +1,31 @@
+## MNIST Examples
+
+Here are two MNIST classifiers implemented in PyTorch.
+The first one is implemented in pure PyTorch, but isn't easy to scale.
+The second one is using [Lightning Lite](https://pytorch-lightning.readthedocs.io/en/stable/starter/lightning_lite.html) to accelerate and scale the model.
+
+#### 1. Image Classifier with Vanilla PyTorch
+
+Trains a simple CNN over MNIST using vanilla PyTorch. It only supports singe GPU training.
+
+```bash
+# CPU
+python image_classifier_1_pytorch.py
+```
+
+______________________________________________________________________
+
+#### 2. Image Classifier with Lightning Lite
+
+This script shows you how to scale the pure PyTorch code to enable GPU and multi-GPU training using [Lightning Lite](https://pytorch-lightning.readthedocs.io/en/stable/starter/lightning_lite.html).
+
+```bash
+# CPU
+lightning run model image_classifier_2_lite.py
```

Comment:
This does not work yet, no? Shouldn't it be changed later when it works?

Reply:
But the PR is here #15506. I don't think it matters a lot in which order to review it. It is pretty self-explanatory or not?

Reply:
Ah right. Because you are just adding it to lightning-app directly. I thought we wouldn't do it yet and needed to refactor the cli as we discussed in a call. Resolving